composer.json

```diff
@@ -120,7 +120,7 @@
         "drupal/page_manager": "1.0.0-alpha24",
         "drupal/panelizer": "3.0.0-alpha2",
         "drupal/scheduled_updates": "1.0.0-alpha5",
-        "drupal/workbench_moderation": "1.1.0",
+        "drupal/workbench_moderation": "^1.0",
```

Comment:
Unfortunately this won't do, because we are still patching Workbench Moderation. When we're patching anything, we have to pin to a specific version as a matter of course. So this should be 1.2.0.